docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -10,88 +10,89 @@ While we're going to debug these issues 'manually' here, workflows (which you'll
 
 .. code:: console
 
-    verdi computer list
+    verdi computer configure local localhost
+
 
-Before the computer can actually be used, we need to actually configure it for a specific user, which allows to connect to that computer.
-Imagine for example that you setup a remote cluster that needs to be accessed over SSH with certain credentials.
-To configure a computer, execute the command:
+Your ``localhost`` computer should now show up in ``verdi computer list``.
+Before proceeding, test that it works:
 
 .. code:: console
 
-    verdi computer configure local <COMPUTER>
+    verdi computer test localhost
 
-replacing ``<COMPUTER>`` with the name of the computer you want to configure, for example ``localhost``.
-This will interactively ask for the configuration details, which for a computer using local transport, such as the localhost, is just the one.
 
-.. note::
+Code setup
+----------
 
-    The command we use here is ``verdi computer configure local`` because the ``localhost`` was configured to use ``local`` transport.
-    If you create a computer that requires ``ssh`` transport, the command to configure connection details would be ``verdi computer configure ssh``.
-
-Now that we have our localhost set up, we can configure the ``pw.x`` executable that we will be running.
-As with the computer, for the virtual machine that comes with the tutorial, you can download the configuration file :download:`here <include/configuration/computer.yml>` and it should contain:
+Now that we have our localhost set up, let's configure the ``Code``, namely the ``pw.x`` executable.
+As with the computer, we have prepared a configuration file for you to :download:`download <include/configuration/computer.yml>`.
+This is its content:
 
 .. literalinclude:: include/configuration/code.yml
     :language: yaml
 
-.. warning::
-
-    The contents of this configuration file will most likely only work for the virtual machine that comes with this tutorial.
-    If you are following this tutorial on your own AiiDA installation, you will have to adapt it yourself.
-    Instructions on how to do this are explained in :ref:`this appendix<appendix_computer_code_setup>`.
-    In addition, you will need a compiled version of Quantum ESPRESSO on your computer.
-
-After you have created or downloaded the file to your local working environment, call the command:
+Once you have the configuration file in your local working environment, set up the code:
 
 .. code:: console
 
     verdi code setup --config code.yml
 
+.. warning::
+
+    The configuration should work for the virtual machine that comes with this tutorial.
+    If you are following this tutorial in a different environment, you will need to install Quantum ESPRESSO and adapt the configuration to your needs, 
+    as explained in :ref:`this appendix<appendix_computer_code_setup>`.
+
 Similar to the computers, you can list all the configured codes with:
```

Comment:
You removed the `verdi computer list` explanation before though

Reply:
it was still there but in the text. moved it back to .. code::